BlocNotes/BlocNotes/DetailNoteViewController.m

```diff
@@ -0,0 +1,97 @@
+//
+//  DetailNoteViewController.m
+//  BlocNotes
+//
+//  Created by Eddy Chan on 6/26/16.
+//  Copyright © 2016 chan-e. All rights reserved.
+//
+
+#import "DetailNoteViewController.h"
+#import "CoreDataStack.h"
+#import "Note.h"
+
+@interface DetailNoteViewController () <UITextViewDelegate>
+
+@property (weak, nonatomic) IBOutlet UILabel *placeholderTextLabel;
+@property (weak, nonatomic) IBOutlet UITextView *bodyTextView;
+
+@end
+
+@implementation DetailNoteViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    self.navigationItem.backBarButtonItem = [[UIBarButtonItem alloc]
```

Comment:
I saw the wireframes for this user story, and the back button had no title. Do you want me to keep the default back button title i.e. "< BlocNotes" ?